doc/media/openage/terrain_format_spec.md

```diff
@@ -0,0 +1,341 @@
+# Terrain Format Specification
+
+**Format Version:** 1
+
+The openage terrain format is a plaintext configuration file format for defining
+terrain textures. It tells the openage renderer which image resources it has to load
+and how textures in these resources should be displayed.
+
+All attributes start with a defined keyword followed by parameter values. Some
+parameters have default values and are optional. The preferred file extension is
+`.terrain`.
+
+
+## Quick Reference
+
+```
+# This is a terrain configuration file
+# comments start with # and are ignored
+
+# file version
+version <version_no>
+
+# source image definitions
+imagefile <image_id> <filename>
+
+# selection of blendomatic borders
+blendmask <blend_id> <filename> <priority>
+
+# How much the texture is scaled down at default zoom level
+scalefactor <factor>
+
+# layer and animation definitions
+# layers defined first will be overdrawn by later definitions
+layer <id> mode=off  position=<int>
+layer <id> mode=loop position=<int> time_per_frame=<float> replay_delay=<float>
+
+# definition of a terrain frames
+# these are iterated for an animation
+frame <layer_id> <image_id> <xpos> <ypos> <xsize> <ysize> blend_id=<int>
```

Comment:
why `blend_id` with keyword arg? also, which of the tiles in a blendmask to take?

Reply:
`frame` is the full terrain texture in this case. Blending can be optional, that's why `blend_id` is an optional keyworded parameter. The tiles to take would depend on the hardcoded blendomatic algorithm.